python/lsst/ts/ledprojector/led_object.py

```diff
@@ -0,0 +1,189 @@
+# This file is part of ts_ledprojector.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["LedObject"]
+import asyncio
+import inspect
+
+
+from lsst.ts import salobj, utils
+from lsst.ts.idl.enums.run_ledprojector import (
+    LEDBasicState,
+)
+from .labjack_interface import LabJackInterface
+from .led_object import LEDObject
+from .mock_labjack_interface import MockLabJackInterface
+
+# DRA-CN024D05 SSR requires 3.0VDC minimum turn-on voltage, up to 12 VDC max
+# 1.0VDC minimum turn-off voltage
+# all ports should be digital IO so write 1 for on/0 for off
+# Values to write to labjack for operating LED
+LED_OFF_SETTING = 0
+LED_ON_SETTING = 1
+
+class LEDObject:
```

Comment:
Not sure I like `LEDObject`. Consider renaming it to something like `LEDManager` or `LEDController`.